tests/internal/e2e/test/specs/vcast_build_env_failure_different_paths.test.ts

```diff
@@ -0,0 +1,210 @@
+
+        subprogram = await findSubprogram(
+          "ENV_01",
+          vcastTestingViewContentSection
+        );
+        if (subprogram && !(await subprogram.isExpanded()))
+          await subprogram.expand();
+
+        subprogram = await findSubprogram(
+          "ENV_03",
+          vcastTestingViewContentSection
+        );
+        if (subprogram && !(await subprogram.isExpanded()))
+          await subprogram.expand();
+      }
+    }
+  });
+
+  it("should change release to 24", async () => {
+    // Release 24
+    const vcastRoot = path.join(process.env.HOME, "vcast");
+    const newVersion = "release24";
+    const release24Path = path.join(vcastRoot, newVersion);
+
+    await updateTestID();
+    const workbench = await browser.getWorkbench();
+    const activityBar = workbench.getActivityBar();
+    const explorerView = await activityBar.getViewControl("Explorer");
+    await explorerView?.openView();
+
+    // Put in release 24 path in settings
+    const settingsEditor = await workbench.openSettings();
+    const unitTestLocationSetting = await settingsEditor.findSetting(
+      "Vectorcast Installation Location",
+      "Vectorcast Test Explorer"
+    );
+    await unitTestLocationSetting.setValue(release24Path);
+
+    await bottomBar.toggle(true);
+    const outputView = await bottomBar.openOutputView();
+
+    // Ignore ENV_01
+    await browser.waitUntil(
+      async () =>
+        (await outputView.getText())
+          .toString()
+          .includes("Ignoring environment"),
+      { timeout: TIMEOUT }
+    );
+    // Build ENV_02
+    await browser.waitUntil(
+      async () =>
+        (await outputView.getText())
+          .toString()
+          .includes("Processing environment"),
+      { timeout: TIMEOUT }
+    );
+    // Ignore ENV_03
+    await browser.waitUntil(
+      async () =>
+        (await outputView.getText())
+          .toString()
+          .includes("Ignoring environment"),
+      { timeout: TIMEOUT }
+    );
+    // Build ENV_04
+    await browser.waitUntil(
+      async () =>
+        (await outputView.getText())
+          .toString()
+          .includes("Processing environment"),
+      { timeout: TIMEOUT }
+    );
+
+    // Open Testing
+    const testingView = await activityBar.getViewControl("Testing");
+    await testingView?.openView();
+
+    let vcastTestingViewContent: ViewContent;
+    let subprogram: TreeItem;
+
+    // Iterate through Testing and try to expand builded Envs (2 & 4)
+    vcastTestingViewContent = await getViewContent("Testing");
+    for (const vcastTestingViewSection of await vcastTestingViewContent.getSections()) {
+      if (!(await vcastTestingViewSection.isExpanded())) {
+        await vcastTestingViewSection.expand();
+      }
+
+      // Expand envs if found
+      for (const vcastTestingViewContentSection of await vcastTestingViewContent.getSections()) {
+        await vcastTestingViewContentSection.expand();
+
```

Comment:
Can you try to reduce code repetition, in particular for expanding multiple environments. Maybe you can write a helper function for this or extend an existing function

Reply:
Reduced code repetiotion in 7c690c6fda33b48239ce5fb7aeb15024a3df53d6.